### ISSUE TITLE
ChromeDev-Tools Optimization

### ISSUE THREAD

**Issue by andre-dietrich** [LiaScript/LiaScript#49]
LiaScript should be checked to fulfill most issues from ChromeDev-Tools in terms of Performace, Best Practices, SEO. Web-Accessibility is a separate point and requires more effort ...
